test/functional/test_framework/test_node.py

```diff
@@ -119,6 +131,7 @@ def stop_node(self):
             self.stop()
         except http.client.CannotSendRequest:
             self.log.exception("Unable to stop node.")
+        self.p2ps = []
```

Comment:
Probably better to do `del self.p2ps[:]` to make sure connections are freed if there are other references to the list. Also would make it clearer that p2ps is an existing member.